apps/src/blockly/googleBlocklyWrapper.js

```diff
@@ -398,8 +398,8 @@ function initializeBlocklyWrapper(blocklyInstance) {
     return false;
   };
 
-  blocklyWrapper.Input.prototype.setStrictCheck = function (check) {
-    return this.setCheck(check);
+  blocklyWrapper.Block.prototype.setStrictOutput = function (isOutput, check) {
```

Comment:
Where was `setStrictOutput` defined prior to this update? I don't see it in mainline?